### PR TITLE
Release 0.16

### LINKED ISSUE #1123: Create releases more often
Simple request, please create releases more often.

0.15 is already over a year old and when adding 0.16-SNAPSHOT to Maven, on every build one can never be sure what version to get and what (potentially breaking) changes.

If you don't want to create final releases, at least make it possible to use a specific snapshot by date or month. Afaik it's possible to configure Nexus to not overwrite deployments, but keep all with a timestamp.

### PR DESCRIPTION
**Pull request by TheCookieLab**
Fixes #1123

Changes proposed in this pull request:
- Updated version numbers to 0.16

- [X] added an entry with related ticket number(s) to the unreleased section of `CHANGES.md`

### CODE REVIEW

**Comment by TheCookieLab** [comment]
Looking for a reviewer. You (yes, you) please step up!

File: pom.xml [hunk 1]
```diff
@@ -52,7 +52,7 @@
 		<connection>scm:git:git://github.com/ta4j/ta4j.git</connection>
 		<developerConnection>scm:git:git@github.com:ta4j/ta4j.git</developerConnection>
 		<url>http://github.com/ta4j/ta4j</url>
-		<tag>0.14</tag>
```
Comment (francescopeloi) [inline pom.xml:55]: was this forgotten in the previous release?

**Comment by TheCookieLab** [comment]
It must have been

**Review by francescopeloi — APPROVED** [review]
LGTM

**Pull request merged**